pilot/pkg/networking/core/tracing_test.go

```diff
@@ -46,6 +46,7 @@ func TestConfigureTracingExhaustiveness(t *testing.T) {
 func TestConfigureTracing(t *testing.T) {
 	clusterName := "testcluster"
 	authority := "testhost"
+	defaultZipkinEndpoint := "/api/v2/spans"
```

Comment:
can you move this to a const, so that it could be shared in two changed files.

Reply:
Done